src/routes/community.md

```diff
@@ -17,6 +17,7 @@
 * <OutboundLink  href="https://github.com/nftstorage/ucan.storage">ucan.storage</OutboundLink> is a UCAN implementation for storage on <OutboundLink href="https://web3.storage/">web3.storage</OutboundLink>
 and <OutboundLink href="https://nft.storage/">nft.storage</OutboundLink>.
 * <OutboundLink  href="https://subconscious.substack.com/">Subconscious</OutboundLink> is a distributed note taking tool that <OutboundLink href="https://subconscious.substack.com/p/layered-protocols">uses UCAN as part of its layered protocol design</OutboundLink>. They are also the authors of the <OutboundLink  href="https://github.com/cdata/rs-ucan">Rust UCAN library rs-ucan</OutboundLink>.
+* <OutboundLink href="https://fileverse.io/">Fileverse</OutboundLink> is a file-sharing and collaboration dApp that uses UCAN to authorize users based on DIDs stored in smart contracts. Their upcoming Fileverse Portal will allow users manage authorzation client side.
```

Comment:
small typo

```suggestion
* <OutboundLink href="https://fileverse.io/">Fileverse</OutboundLink> is a file-sharing and collaboration dApp that uses UCAN to authorize users based on DIDs stored in smart contracts. Their upcoming Fileverse Portal will allow users to manage authorzation client side.
```

Reply:
Fixed type of "authorzation" -> "authorization"

Reply:
Thanks! Good catch 🙌